ci/fetch-artifacts.sh

```diff
@@ -2,48 +2,92 @@
 
 # script to fetch artifacts from Win,OSX CI to Travis,
 # where all 3 files are deployed to GitHub for release. 
+# The curent pwd should be build/scripts when running this script.
 
-WINDOWS_ARTIFACT_URL="https://ci.appveyor.com/api/buildjobs/by4cpqmul79tp9ag/artifacts/build/scripts/htm_core-6866bd13ece35608313709e82c29b2260ee7be73-windows64.zip" #FIXME make the URLs automated, needs API?
-OSX_ARTIFACT_URL="https://3238-118031881-gh.circle-artifacts.com/0/tmp/uploads/htm_core-7b7b53c798b3795088ce772e96c6994068fc1302-darwin64.tar.gz"
+
+export VERSION=`cat ../../VERSION`
+echo "Released version: $VERSION"
+echo
+
+export TRAVIS_FILE="htm_core-$VERSION-linux64.tar.gz"
+
+#WINDOWS_ARTIFACT_URL="https://ci.appveyor.com/api/buildjobs/by4cpqmul79tp9ag/artifacts/build/scripts/htm_core-6866bd13ece35608313709e82c29b2260ee7be73-windows64.zip?branch=master" #FIXME make the URLs automated, needs API?
+#OSX_ARTIFACT_URL="https://3238-118031881-gh.circle-artifacts.com/0/tmp/uploads/$CIRCLE_FILE"
+
+
+echo
+echo "Fetching Windows artifacts:"
+export APPVEYOR_TOKEN="v2.5tido3qf80n33le5bi3e"  # token by David Keeney.  TODO: replace with project token.  Needs Admin permissions.
```

Comment:
where do I set this? Is it an API token at GH, or some token generated in AppV?

Reply:
This is the token for Appveyor.  Someone with admin privs should login to AppV and create a general token for the htm-community and put it here.

The toke you see there is a personal token I created for numenta-ci because it does not think I am a member of htm-community.   So it did not work.

Reply:
Someone also needs to get a general token for htm-community on CircleCI and put that in here as well.

Reply:
> with admin privs should login to AppV and create a general token for the htm-community and put it here

i just skimmed through the settings, didn't find a way in AppV to make such token, could you please give me a hand there? 
Also, is it safe to have such token public? (can we use sth like `secret: ...` in CircleCI?)

Reply:
Ready in https://github.com/htm-community/htm.core/pull/639#issuecomment-522631073